WordPress/src/main/java/org/wordpress/android/ui/prefs/SiteSettingsFragment.java

```diff
@@ -384,9 +382,8 @@ public boolean onPreferenceClick(Preference preference) {
         } else if (preference == mCategoryPref || preference == mFormatPref) {
             return !shouldShowListPreference((DetailListPreference) preference);
         } else if (preference == mExportSitePref) {
-            showExportContentDialog();
```

Comment:
I am sorry if I missed a conversation about this. Why are we removing this? Also, if we are removing it, I think we should also remove the if statement: `else if (preference == mExportSitePref) {` since it's empty now.